src/libraries/System.DirectoryServices/src/System/DirectoryServices/DirectoryServicesCOMException.cs

```diff
@@ -56,17 +57,19 @@ internal static Exception CreateFormattedComException(int hr)
         internal static Exception CreateFormattedComException(COMException e)
         {
             // get extended error information
-            const int ErrorBufferLength = 256;
-            Span<char> errorBuffer = stackalloc char[ErrorBufferLength];
-
-            Span<char> nameBuffer = Span<char>.Empty;
+            char[] errorBuffer = ArrayPool<char>.Shared.Rent(256);
```

Comment:
There probably isn't a lot of value to using the `ArrayPool<char>` here. I would just allocate the array and avoid the management below.

Reply:
Changed in 50aa014cc1207fa27b7493cd7ec290e46c031b4f